### PR TITLE
For #1444 Exposed title in GeckoEngineSession - Nightly

### PR DESCRIPTION
**Pull request by mcarare**
Added missing parameter element.title

---

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt [hunk 1]
```diff
@@ -516,7 +516,7 @@ class GeckoEngineSession(
             screenY: Int,
             element: GeckoSession.ContentDelegate.ContextElement
         ) {
-            val hitResult = handleLongClick(element.srcUri, element.type, element.linkUri)
+            val hitResult = handleLongClick(element.srcUri, element.type, element.linkUri, element.title)
```
Comment (pocmo) [inline components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt]: Is this only available in 71.0 (`browser-engine-gecko-nightly`) for now?

Comment (mcarare) [reply components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt]: No, it was already  available in the others, missed it in here.

Comment (pocmo) [reply components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt]: Ah, okay :)

**Pull request merged**